vmm_tests/vmm_tests/tests/tests/x86_64.rs

```diff
@@ -323,19 +323,17 @@ fn configure_for_sidecar(
         })
 }
 
-// Sidecar currently requires the dev kernel build.
-//
 // Use UEFI so that the guest doesn't access the other APs, causing hot adds
 // into VTL2 Linux.
 //
 // Sidecar isn't supported on aarch64 yet.
 #[vmm_test(
-    openvmm_openhcl_uefi_x64(none) [LATEST_STANDARD_DEV_KERNEL_X64],
-    // TODO: debug why boot is failing  hyperv_openhcl_uefi_x64(none) [LATEST_STANDARD_DEV_KERNEL_X64],
+    openvmm_openhcl_uefi_x64(none) [LATEST_STANDARD_X64],
```

Comment:
This is the default, so you can just delete all the IGVM-related handling from these tests.